src/prefect/cli/server.py

```diff
@@ -230,24 +230,27 @@ async def start(
     app.console.print("\n")
 
     try:
+        command = [
+            sys.executable,
```

Comment:
any reason for the switching logic would not be contained in `get_sys_executable`?

Reply:
`get_sys_executable` always quotes on Windows, which we don't want here. That seems to be necessary with the internal `run_process` util, but breaks when using `anyio` process utilities directly.